doc/nrf/ug_thread_tools.rst

```diff
@@ -37,22 +37,242 @@ Thread Border Router
 Thread Border Router is a specific type of Border Router device that provides connectivity from the IEEE 802.15.4 network to adjacent networks on other physical layers (such as Wi-Fi or Ethernet).
 Border Routers provide services for devices within the IEEE 802.15.4 network, including routing services for off-network operations.
 
+`OpenThread Border Router`_ (OTBR) released by Google is an open-source implementation of the Thread Border Router recommended to use for development purpose.
```

Comment:
Do we even need this whole "`OpenThread Border Router`_ (OTBR) released by Google ..." sentence? Few lines below we already have very similar sentence: "For development purposes, you can use `OpenThread Border Router`_ ...", so maybe it would be better to remove this part from here and rephrase the part below, as follows:
"For development purposes, you can use the `OpenThread Border Router`_, an open-source Border Router implementation released by Google, that you can set up either on your PC using Docker or on Raspberry Pi."?

Reply:
@kkasperczyk-no I removed it in the latest push.